frontend/src/features/public-form/components/FormPaymentPreview/FormPaymentPreview.tsx

```diff
@@ -27,6 +27,7 @@ export const FormPaymentPreview = ({
   const sectionColor = useSectionColor(colorTheme)
   const emailFieldSchema: VerifiableEmailFieldSchema = {
     ...(getFieldCreationMeta(BasicField.Email) as EmailFieldBase),
+    title: 'Email Address',
     _id: PAYMENT_CONTACT_FIELD_ID,
     description: 'Proof of payment will be sent to this email',
```

Comment:
JTC wanted to change this to "For delivery of invoice", right? What is our stance on this? (Not sure if I missed some discussion in office? 😅)